### DISCUSSION TITLE
Universal VAE Node?

### DISCUSSION THREAD

**Discussion by poisenbery** [Comfy-Org/ComfyUI#1922] [Ideas]
A lot of nodes require a VAE input.
However, this is something that I literally NEVER change, at all, ever, during my workflow.

Now, this might be just a ME problem, but it becomes rather tedious to have to connect a VAE to dozens of nodes across the screen. It causes my workflows to be rather cluttered because of the extra connection going across the entire screen.

I think it would be useful to have a node or a setting that can force the VAE for any node that has the input, without requiring the user to manually connect the VAE node.

**Comment by poisenbery** [comment]
I might need too look at the code from @ltdrdata Global Seed Node from [This](https://github.com/ltdrdata/ComfyUI-Inspire-Pack) because it's the only node I have that can influence other nodes without being connected to them.

**Comment by Erehr** [comment]
[cg-use-everywhere](https://github.com/chrisgoringe/cg-use-everywhere) is what you need. It will 'broadcast' anything (in your case vae) to any unconnected matching input

**Reply by poisenbery** [reply]
Perfect, thank you!

**Comment by poisenbery** [comment]
> [cg-use-everywhere](https://github.com/chrisgoringe/cg-use-everywhere) is what you need. It will 'broadcast' anything (in your case vae) to any unconnected matching input
